solutions/usaco-915.mdx

```diff
@@ -0,0 +1,58 @@
+---
+id: usaco-915
+source: USACO Bronze 2019 February
+title: Sleepy Cow Herding
+author: Sathvik Chundru
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_herding_bronze_feb19.html)
+
+<LanguageSection>
+ 
+<JavaSection>
+
+```java
+import java.util.*;
+import java.io.*;
+class Main {
+	public static void main(String[] args) throws IOException {
+		Scanner sc = new Scanner(new File("herding.in"));
+		PrintWriter pw = new PrintWriter(new File("herding.out")); 
+		int a = sc.nextInt();
+		int b = sc.nextInt();
+		int c = sc.nextInt();
+		int temp;
+		if (a > b) {
+			temp = a;
+			a = b;
+			b = temp;
+		}
+		if (b > c) {
+			temp = b;
+			b = c;
+			c = temp;
+		}
+		if (a > b) {
+			temp = a;
+			a = b;
+			b = temp;
+		}
```

Comment:
minor detail, but can we:

- add a comment to the top of this block that says all we're doing is arranging a b c to be in ascending order
- add a comment that all the temp = a / a = b / b = temp code does is swap the variables a and b
- add a blank line around this block to better separate this section from the rest of the code

(unless you think some of these are unnecessary?)

Reply:
minor detail based on preference but i have added these changes

Reply:
it would be nice to make a swap function instead